File: 03 Berechnungen/Go/Aufgabe3.go
package main

import "fmt"

var a float32 = 7.1

func main() {
	fmt.Println("a =", a)

	a++
	fmt.Println("a++; (a = a + 1) a =", a)

	a--
	fmt.Println("a--; (a = a - 1) a =", a)

	a += 6
	fmt.Println("a += 6; (a = a + 6) a =", a)

	a -= 6
	fmt.Println("a -= 6; (a = a - 6) a =", a)

	a *= 6
	fmt.Println("a *= 6; (a = a * 6) a =", a)

	a /= 6 // In Go geht das!
	fmt.Println("a /= 6; (a = a / 6) a =", a)

	fmt.Println("a % 2 geht nicht bei Kommazahlen!")
}

File: 03 Berechnungen/Pascal/Aufgabe3.pas
Program Aufgabe1;

{$codepage utf8}

Var A: Real = 7.1;

Begin
    WriteLn('A = ', A:1:2);

//  inc(A);
    WriteLn('inc(A); geht nicht!');

//  dec(A);
    WriteLn('dec(A); geht nicht!');

    A += 6;
    WriteLn('A += 6; (A = A + 6) A = ', A:1:2);

    A -= 6;
    WriteLn('A -= 6; (A = A - 6) A = ', A:1:2);

    A *= 6;
    WriteLn('A *= 6; (A = A * 6) A = ', A:1:2);

    A /= 6;
    WriteLn('A /= 6; (A = A / 6) A = ', A:1:2);

    WriteLn('A % 2 geht nicht bei Kommazahlen!');
End.